cli/command/container/formatter_stats.go

```diff
@@ -29,6 +31,7 @@ type StatsEntry struct {
 	Container        string
 	Name             string
 	ID               string
+	AutoRange		 string
```

Comment:
nit: I think you should configure your text editor/IDE to run a `go fmt` each time you save a go file.

Reply:
Will do it ASAP